banyand/stream/elements.go

```diff
@@ -150,6 +152,7 @@ type elements struct {
 	timestamps  []int64
 	elementIDs  []uint64
 	tagFamilies [][]tagValues
+	indexedTags []map[string]map[string]struct{}
```

Comment:
How about adding a field `indexed` in `tagValue` to indicate whether the tag is indexed? The benefit of such a path is that you don't have to add a new parameter to several functions.